frontend/src/app/containers/App.js

```diff
@@ -70,7 +70,7 @@ class App extends Component {
     if (window.ga && ga.loaded) {
       const data = dataIn || {};
       data.hitType = type;
-      data.page = (pathname === '/') ? pathname : '/' + pathname;
+      data.location = window.location;
```

Comment:
> `window.location` would also contain the host. We want `window.location.pathname`, right?

We need the host so GA can filter pings to the dev/stage/prod profiles. Previously the host was being sent implicitly through the `location` field. We're overriding location here since GA wasn't picking up the path correctly.